docs/release-notes/1.3.0.asciidoc

```diff
@@ -0,0 +1,64 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.3.0]]
+== {n} version 1.3.0
+
+[[breaking-1.3.0]]
+[float]
+=== Breaking changes
+
+* Enforce a strict upgrade order between stack components {pull}3537[#3537] (issues: {issue}2600[#2600], {issue}3535[#3535])
+* Add set-default-security-context flag to the operator {pull}3342[#3342] (issue: {issue}2791[#2791])
+
+
+[[feature-1.3.0]]
+[float]
+=== New features
+
+* Make timeouts configurable {pull}3782[#3782] (issue: {issue}684[#684])
+* Support Elasticsearch volumes expansion {pull}3752[#3752] (issues: {issue}325[#325], {issue}3757[#3757], {issue}3767[#3767])
+* Add DeploymentStrategy to Beats CRDs {pull}3633[#3633] (issue: {issue}3485[#3485])
+* Add operator election support {pull}3632[#3632] (issue: {issue}709[#709])
+
+[[enhancement-1.3.0]]
+[float]
+=== Enhancements
+
+* Add license info to telemetry {pull}3859[#3859]
+* Validate whether the storage class allows volume expansion before resizing volumes {pull}3795[#3795] (issue: {issue}3767[#3767])
+* Add external IP addresses to certificate {pull}3791[#3791] (issue: {issue}910[#910])
+* Support a UBI-only mode in the operator {pull}3789[#3789]
+* Add CRD usage reporting {pull}3784[#3784]
+* Refactor Helm chart {pull}3711[#3711]
```

Comment:
Shall I re-title this PR to say "Refactor Helm chart for public usage" and re-label it as a feature?

Reply:
I went ahead and did that already.